src/Symfony/Component/Form/Extension/Core/Type/DateType.php

```diff
@@ -288,6 +288,14 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('years', 'array');
         $resolver->setAllowedTypes('months', 'array');
         $resolver->setAllowedTypes('days', 'array');
+
+        $resolver->setNormalizer('html5', function (Options $options, $html5) {
+            if ($html5 && 'single_text' === $options['widget'] && self::HTML5_FORMAT !== $options['format']) {
+                @trigger_error(sprintf('Using a custom format when the "html5" option of the %s is enabled is deprecated since Symfony 4.2.', self::class), E_USER_DEPRECATED);
```

Comment:
will this code path throw in Symfony 5? or will we just ignore the property?
if throw => the code should be added as a commented line
if ignore => might end up being confusing